### ISSUE TITLE
Document encryption not working?

### ISSUE THREAD

**Issue by auv** [glushchenko/fsnotes#1713]
### Description

I am using version 4.9.6. The version of the MAC system is 14.2.1. I have an extremely important work diary that has been recorded for 2 years. When I tried to encrypt the document, the pop-up dialog box did not allow me to enter the password twice to confirm that the password I entered was the one I wanted to input. After I entered it, the document was locked. But no matter how I try, I can't unlock my diary file again. Please help me, thank you. I really need this work diary file, it is everything to me in my work.

### FSNotes version

4.9.6

### macOS/iOS version

 14.2.1

**Comment by gingerbeardman** [comment]
Please download the latest version. Currently 6.7.0

https://github.com/glushchenko/fsnotes/releases/latest

**Comment by auv** [comment]
> Please try the latest version. Currently 6.7.0
> 
> https://github.com/glushchenko/fsnotes/releases/latest
> 
> I'm not sure what you are describing. So the password has been the same and working for 2 years, and only recently it will not let you unlock it?


Thank heavens, gingerbeardman! I've recovered the etp file containing my 2-year work diary that I described as being undecryptable. I don't know where things went wrong. Here's the path where I encountered the issue: 1. Using version 4.9.6, I encrypted a document by entering the password (just once). 2. When I clicked the decrypt button and entered the same password, it failed to decrypt. I thought for a moment I was going to lose this document. Here's how I solved the issue: I changed the extension of that etp file to a .txt file, then dragged the file into the Visual Studio Code software for viewing, and it was completely displayed! Thank heavens! This phenomenon suggests that the ept file wasn't actually encrypted, right? By a stroke of luck, at least my document was saved, thank you, gingerbeardman.
